http-gateway/swagger.yaml

```diff
@@ -19,6 +19,45 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/CloudConfiguration'
+  '/api/v1/oauth/callback':
+    get:
+      tags:
+        - 'Configuration'
+      summary: 'Converts OAuth redirect response to body'
+      description: |
+        Converts oauth redirect response to body as JSON. It used for getting device onboarding authorzation code as redirect.
+      responses:
+        200:
+          description: OAuth response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OAuthResponse'
+        400:
+          $ref: '#/components/responses/badRequest'
+        401:
+          $ref: '#/components/responses/unauthorized'
+        403:
+          $ref: '#/components/responses/forbidden'
+    post:
+      tags:
+        - 'Configuration'
+      summary: 'Converts OAuth redirect response to body'
+      description: |
+        Converts oauth redirect response to body as JSON. It used for getting device onboarding authorzation code as redirect.
```

Comment:
Same as above. 

(Post and get do the same thing?)

Reply:
Fixed. Yes, it just converts redirect response to JSON body.